.github/workflows/staging-build-tar.yml

```diff
@@ -459,3 +459,50 @@ jobs:
           working-directory: kibana/plugins/anomaly-detection-kibana-plugin
           command: yarn cy:run-with-security --config baseurl=http://localhost:5601
     
+  Test-Perf-Analyzer-No-Security:
+    runs-on: ubuntu-18.04
+    name: Test-Perf-Analyzer-No-Security
+    strategy:
+      matrix:
+        java: [12]
+    steps:
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Set up AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Checkout opendistro-build
+        uses: actions/checkout@v1
+      - name: Checkout perf analyzer RCA package
+        uses: actions/checkout@v1
+        with:
+          repository: opendistro-for-elasticsearch/performance-analyzer-rca
+          # TODO: Change to a tag once new tags have been published
+          ref: master
```

Comment:
Not only you directly checkout master in the yml, but also you did not retrieve the tag to checkout from there.

- name: Retrieve plugin tags
        run:  echo ::set-env name=p_tag_pa::$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/performance-analyzer)

- name: Retrieve plugin tags
        run:  echo ::set-env name=p_tag_pa_rca::$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/performance-analyzer=rca)